src/plone/restapi/tests/test_serializer_converters.py

```diff
@@ -121,11 +127,11 @@ def test_dict(self):
                                            'baz': 3}))
         self.assertEquals('{"foo": "bar"}',
                           json.dumps(json_compatible({'foo': 'bar'})))
-        self.assertIsInstance(json_compatible({'foo': 'bar'}.keys()[0]),
-                              unicode,
+        self.assertIsInstance(json_compatible(list({'foo': 'bar'}.keys())[0]),
+                              six.text_type,
                               'Dict keys should be converted recursively.')
-        self.assertIsInstance(json_compatible({'foo': 'bar'}.values()[0]),
-                              unicode,
+        self.assertIsInstance(json_compatible(list({'foo': 'bar'}.values())[0]),
```

Comment:
this is inefficient in Python 2; it should be written:

```python
from six import itervalues
self.assertIsInstance(json_compatible(list(itervalues({'foo': 'bar'}))[0]),
```

`{'foo': 'bar'}` should be a variable.